lib/features/autentication/signup_page.dart

```diff
@@ -0,0 +1,71 @@
+import 'package:flutter/material.dart';
+import 'package:firebase_auth/firebase_auth.dart';
+
+class SignupPage extends StatefulWidget {
+  const SignupPage({super.key});
+
+  @override
+  SignupPageState createState() => SignupPageState();
+}
+
+class SignupPageState extends State<SignupPage> {
```

Comment:
Don't use `StatefulWidget` across the entire project. It is good to know how they function, I am just pushing you to learn what we will use in the projects.